slim/Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN set -x \
     && cd / && rm -rf /znc-src
 
 COPY docker-entrypoint.sh /
-
+RUN chmod +x /docker-entrypoint.sh
```

Comment:
Maybe you accidentally cleared the executable bit from the file in your git checkout before building the image? Or some strange filesystem you use did that for you?